components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -30,6 +35,12 @@ export function getTeamSettingsMenu(params: { team?: Team; billingMode?: Billing
             link: [`/sso`],
         });
     }
+    if (orgGitAuthProviders) {
```

Comment:
✅ check: `orgGitAuthProviders` is considered, those this is not automatically available on gitpod.io, only if the feature flag is enabled.